conda/recipes/cugraph/recipe.yaml

```diff
@@ -0,0 +1,116 @@
+# Copyright (c) 2025, NVIDIA CORPORATION.
+schema_version: 1
+
+context:
+  version: ${{ env.get("RAPIDS_PACKAGE_VERSION") }}
+  minor_version: ${{ (version | split("."))[:2] | join(".") }}
+  cuda_version: ${{ (env.get("RAPIDS_CUDA_VERSION") | split("."))[:2] | join(".") }}
+  cuda_major: '${{ (env.get("RAPIDS_CUDA_VERSION") | split("."))[0] }}'
+  date_string: '${{ env.get("RAPIDS_DATE_STRING") }}'
+  py_version: ${{ env.get("RAPIDS_PY_VERSION") }}
+  py_buildstring: ${{ py_version | version_to_buildstring }}
+  head_rev: ${{ git.head_rev(".")[:8] }}
+
+package:
+  name: cugraph
+  version: ${{ version }}
+
+source:
+  path: ../../..
+
+build:
+  string: cuda${{ cuda_major }}_py${{ py_buildstring }}_${{ date_string }}_${{ head_rev }}
+  files:
+    exclude:
+      - '*libarrow.so.*gdb.py'
```

Comment:
Where does cuGraph end up with libarrow bits?

Reply:
I'll have to rebuild to find it again, I'll start that going in the background

Reply:
Here's an example of this file getting included in the package: https://github.com/rapidsai/cugraph/actions/runs/14196224989/job/39773072531?pr=4999#step:9:3094

Reply:
Weird. I don't fully understand but I'll roll with it.

Reply:
https://github.com/prefix-dev/rattler-build/issues/979